### ISSUE TITLE
starboard-exporter crashing because of missing permisisons

### ISSUE THREAD

**Issue by sfhl** [giantswarm/starboard-exporter#191]
Hi there,
using the helm template, starboard exporter crashes because of the following error

```
E0213 12:21:46.781018       1 reflector.go:140] pkg/mod/k8s.io/client-go@v0.26.0/tools/cache/reflector.go:169: Failed to watch *unstructured.Unstructured: failed to list *unstructured.Unstructured: endpoints "starboard-exporter" is forbidden: User "system:serviceaccount:security:starboard-exporter" cannot list resource "endpoints" in API group "" in the namespace "security"
```
I guess this  may be easily fixed by moving the following code from `Role` `starboard-exporter-psp` to `ClusterRole` `starboard-exporter` (for example).

```
...
    - apiGroups:       
        - ""           
      resources:       
        - endpoints    
      verbs:           
        - get          
        - list         
        - watch        
...
```

**Comment by stone-z** [comment]
Hi @sfhl, welcome :wave: 
Can you provide more detail about how you installed the exporter? Please double check any adjustments to the RBAC setup specifically.

For security reasons, we wouldn't want to allow the exporter to list endpoints at the cluster scope, but it shouldn't be necessary. The `endpoints` permissions you referenced are not related to the `psp` ClusterRole -- they are part of [this Role](https://github.com/giantswarm/starboard-exporter/blob/main/helm/starboard-exporter/templates/rbac.yaml#L41) which should be deployed in any case.

**Comment by sfhl** [comment]
Hi @stone-z 
sure, i am deploying via helm, with only one modification (`exporter.vulnerabilityReports.targetLabels`). So there is no modification on rbac templates or anythig else. If using the default templates, so the referenced `Role` the exporter crashes with the given error.

Kind regards

**Comment by stone-z** [comment]
Fix is released with [v0.7.2](https://github.com/giantswarm/starboard-exporter/blob/main/CHANGELOG.md#072---2023-02-27)
